securitycenter/assets/main.go

```diff
@@ -0,0 +1,220 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Contains examples calls to Cloud Security Center ListAssets API method.
+
+package main
+
+import (
+	securitycenter "cloud.google.com/go/securitycenter/apiv1beta1"
+	"context"
+	"fmt"
+	"github.com/golang/protobuf/ptypes"
+	"google.golang.org/api/iterator"
+	securitycenterpb "google.golang.org/genproto/googleapis/cloud/securitycenter/v1beta1"
+	"os"
+	"time"
+)
+
+// Lists all assets currently in an organization.
+// [START list_all_assets]
+func ListAllAssets(orgId string) (int, error) {
+	// Instantiate a context and a security service client to make API calls.
+	ctx := context.Background()
+	client, err := securitycenter.NewClient(ctx)
+	if err != nil {
+		fmt.Printf("Error instantiating client %v\n", err)
+		return 0, err
+	}
+
+	listAssetsRequest := securitycenterpb.ListAssetsRequest{}
+	// orgId is the numeric organization ID.  e.g. 01231231
+	listAssetsRequest.Parent = fmt.Sprintf("organizations/%s", orgId)
```

Comment:
Please inline the Parent assignment when building the request.
Something like (untested):

```
req := &securitycenterpb.ListAssetsRequest{
	Parent: fmt.Sprintf("organizations/%s", orgID),
}
```